aictl/cli/main.py

```diff
@@ -196,7 +203,7 @@ def main():
     t2i_parser.add_argument('-c', '--cfg', default='7.5', help='higher values tell the image gen to follow the prompt more closely (default=7.5)', type=float)
     t2i_parser.add_argument('-d', '--denoiser', default='0.7', help='modulate the influence of guidance images on the denoising process (default=0.7)', type=float)
     t2i_parser.add_argument('-b', '--batch-size', default='1', help='number of images per generation', type=int)
-    t2i_parser.add_argument('-o', '--output-path', default='output_t2i.png', help='path for image output when generation is complete')
+    t2i_parser.add_argument('-o', '--output-path', default='output_t2i'+str(utc_time)+'.png', help='path for image output when generation is complete')
```

Comment:
Can you use python format strings here instead of concatenation like so:
```python
default=f'output_t2i-{utc_time}.png'
```
format strings will automatically coerce your variable into a string so there is no need for `str()` here. Do the same for the other commands as well.

Reply:
Thank you for your comment.